website/static/js/fangorn.js

```diff
@@ -1061,12 +1654,35 @@ tbOptions = {
         $(document).on('click', '.fangorn-dismiss', function() {
             tb.redraw();
         });
+        tb.select('#tb-tbody').on('click', function(event){
+            if(event.target !== this) {
+                return;
+            }
+            tb.clearMultiselect();
+            _fangornResetToolbar.call(tb);
+        })
 
         $(window).on('beforeunload', function() {
             if (tb.dropzone && tb.dropzone.getUploadingFiles().length) {
               return 'You have pending uploads, if you leave this page they may not complete.';
             }
         });
+        if(tb.options.placement === 'project-files') {
+            _resizeHeight.call(tb);
+            $(window).resize(function(){
+                _resizeHeight.call(tb);
+            })
+        }
+        $(window).on('keydown', function(event){
+            if (event.keyCode === 27) {
+                _fangornResetToolbar.call(tb);
+            }
+        });
+        $(document).on('keypress', '#createFolderInput', function () {
+            if (tb.pressedKey === 13) {
```

Comment:
Right, but why is treebeard tracking it? Any keypress event can access it through evt.keyPress